### PR TITLE
SSTOOLKIT-132 SSTOOLKIT-133

### PR DESCRIPTION
**Pull request by bertofl90**

### CODE REVIEW

File: docs/xroad_security_server_toolkit_user_guide.md [hunk 1]
```diff
@@ -1607,6 +1611,52 @@ xrdsst csr delete --ss <SECURITY_SERVER_NAME> --key <KEY_ID> --csr <CSR_ID>
...
+#### 4.2.16 Security Server management
+##### 4.2.16.1 List Security Servers
+
+* Access rights: Any role
+
+Listing security servers can be done with:
+```
+xrdsst security-server list
+```
+This command will display the union of all the security servers discover by each security server stored in the configuration file.
```
Comment (bertvi) [inline docs/xroad_security_server_toolkit_user_guide.md]: "all security servers discovered by security server"? Seems confusing. Better to state something like "This command will display all discovered security servers"

File: docs/xroad_security_server_toolkit_user_guide.md [hunk 2]
```diff
@@ -1607,6 +1611,52 @@ xrdsst csr delete --ss <SECURITY_SERVER_NAME> --key <KEY_ID> --csr <CSR_ID>
...
+* MEMBER CLASS security server owner class
+* MEMBER CODE security server owner code
+
+##### 4.2.16.2 List Security Server version
+
+* Access rights: Any role
+
+Listing security server version be done with:
+```
+xrdsst security-server version
```
Comment (bertvi) [inline docs/xroad_security_server_toolkit_user_guide.md]: I think "list-version" would be less confusing

File: xrdsst/controllers/security_server.py [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+                render_data.extend(map(SecurityServerVersionMapper.as_object, render_list))
+            self.render(render_data)
+        BaseController.log_keyless_servers(ss_api_conf_tuple)
+        return render_list
+
+    def remote_list_security_servers(self, ss_api_config):
+        security_servers_api = SecurityServersApi(ApiClient(ss_api_config))
+        try:
+            security_servers = self.parse_security_server_response(security_servers_api.get_security_servers())
+            return list(security_servers)
```
Comment (bertvi) [inline xrdsst/controllers/security_server.py:116]: get_security_servers() already returns a list, so you are creating a list of lists here?

**Review by bertvi — CHANGES_REQUESTED** [review]

File: xrdsst/controllers/security_server.py (hunk 3 above)
Comment (bertofl90) [reply xrdsst/controllers/security_server.py:116]: no, i doesn't return a list, returns other type of generated object, it works fine with the cast to list

**Review by bertvi — APPROVED** [review]

**Pull request merged**